lib/vsc/utils/run.py

```diff
@@ -86,6 +86,48 @@
 SHELL = BASH
 
 
+class CmdList(list):
+    """Wrapper for 'list' type to be used for constructing a list of options & arguments for a command."""
+
+    def __init__(self, cmd=None):
+        """
+        Create CmdList instance to construct command
+
+        :param cmd: actual command to run (first item in list)
+        """
+        super(CmdList, self).__init__()
+        if cmd is not None:
+            if isinstance(cmd, list):
+                super(CmdList, self).extend(cmd)
+            else:
+                super(CmdList, self).append(cmd)
+
+    def add_opts_args(self, items, tmpl_vals=None, allow_spaces=True):
+        """
+        Add options/arguments to command
+
+        :param item: option/argument to add to command
+        :param tmpl_vals: template values for item
+        """
+        if isinstance(items, basestring):
```

Comment:
will break the python3 porting effort

Reply:
We'll fix that when we actually get to porting `run.py` to Python 3 (we don't have a replacement for this yet that works with both Py2 & Py3).